packages/core/useElementHover/index.ts

```diff
@@ -1,13 +1,38 @@
 import type { Ref } from 'vue-demi'
-import { ref } from 'vue-demi'
-import type { MaybeComputedRef } from '@vueuse/shared'
+import { computed, ref } from 'vue-demi'
 import { useEventListener } from '../useEventListener'
+import type { MaybeElementRef } from '../unrefElement'
+import { unrefElement } from '../unrefElement'
+import type { ConfigurableWindow } from '../_configurable'
 
-export function useElementHover(el: MaybeComputedRef<EventTarget | null | undefined>): Ref<boolean> {
+export interface MouseHoverOptions extends ConfigurableWindow {
+  enterDelay?: number
+  leaveDelay?: number
```

Comment:
```suggestion
  delayEnter?: number
  delayLeave?: number
```

Probably can be better discoverable by IDE